### PR TITLE
Dropdown menu to Chakra UI

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR refactors Dropdown Menu to use Chakra UI

demo

### CODE REVIEW

File: assets/src/infrastructure/ui/layout/menu/types.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export interface MenuDividerProps extends React.ComponentProps<typeof MenuDivider> {}
+
+export interface MenuGroupProps extends ChakraMenuGroupProps {}
+
+export interface MenuItemOptionProps extends ChakraMenuItemOptionProps {}
+
+export interface MenuItemProps extends ChakraMenuItemProps {}
+
+export type MenuListProps = ChakraMenuListProps & {};
```
Comment (manzoorwanijk) [inline assets/src/infrastructure/ui/layout/menu/types.ts:21]: `ChakraMenuListProps` is defined as `type`

File: assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/TicketMainMenuItem.tsx [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-	<EspressoDropdownMenu icon={Icon.MORE} label={__('event ticket main menu')}>
-		<EditTicket />
-		<CopyTicket />
-		<TrashTicket />
-	</EspressoDropdownMenu>
-);
+const TicketMainMenuItem: React.FC = () => {
+	const toggleProps = {
+		label: __('ticket main menu'),
+	};
```
Comment (tn3rb) [inline assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/TicketMainMenuItem.tsx:14]: forgot the icon

```suggestion
	const toggleProps = {
		icon: Icon.MORE,
		label: __('ticket main menu'),
	};
```

File: assets/src/infrastructure/ui/inputs/Button.tsx [hunk 3]
```diff
@@ -3,9 +3,15 @@ import { Button as ChakraButton } from '@chakra-ui/core';
 
 import type { ButtonProps } from './types';
 
-const Button: React.FC<ButtonProps> = ({ children, buttonText, ...props }) => {
-	const text = children || buttonText;
-	return <ChakraButton {...props}>{text}</ChakraButton>;
-};
+const Button: React.FC<ButtonProps> = React.forwardRef<typeof ChakraButton, ButtonProps>(
+	({ children, buttonText, ...props }, ref) => {
+		const text = children || buttonText;
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/inputs/Button.tsx:8]: 👍

File: assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/TicketMainMenuItem.tsx (hunk 2 above)
Comment (manzoorwanijk) [reply assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/TicketMainMenuItem.tsx:14]: That's by default set to `Icon.MORE`

**Pull request merged**